src/Components/Nav/NavHeadLogos.js

```diff
@@ -0,0 +1,29 @@
+import React, { Component } from 'react';
+import { BsList } from 'react-icons/bs';
+import { BsBell } from 'react-icons/bs';
```

Comment:
```suggestion
import { BsList, BsBell } from 'react-icons/bs';
```
한번에 import 할 수 있습니다.